eras/conway/impl/test/Test/Cardano/Ledger/Conway/GenesisSpec.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Test.Cardano.Ledger.Conway.GenesisSpec (spec) where
+
+import Cardano.Ledger.BaseTypes (StrictMaybe (..))
+import Cardano.Ledger.Conway
+import Cardano.Ledger.Conway.Genesis (ConwayGenesis (..))
+import Cardano.Ledger.Conway.Governance (Committee (..))
+import Cardano.Ledger.Core
+import Cardano.Ledger.Credential
+import Cardano.Ledger.Crypto (StandardCrypto)
+import Cardano.Ledger.Keys
+import Cardano.Ledger.Slot (EpochNo (..))
+import Data.Aeson hiding (Encoding)
+import Data.Default.Class (Default (def))
+import Data.Map as Map
+import Data.Ratio ((%))
+import Paths_cardano_ledger_conway (getDataFileName)
+import Test.Cardano.Ledger.Common
+import Test.Cardano.Ledger.Core.Utils (unsafeBoundRational)
+
+spec :: Spec
+spec = do
+  describe "Genesis Golden Spec" $ do
+    goldenJSON "test/data/conway-genesis.json"
+
+goldenJSON :: FilePath -> Spec
+goldenJSON fileName = do
```

Comment:
No point in supplying a file name as argument, since that test will only work for that file.
```suggestion
    goldenConwayGenesisJSON

goldenConwayGenesisJSON :: Spec
goldenConwayGenesisJSON = do
  let fileName = "test/data/conway-genesis.json"
```